analyzer/src/main/kotlin/managers/Pip.kt

```diff
@@ -260,6 +260,7 @@ class Pip(
         }
         pip.requireSuccess()
 
+        var authors: SortedSet<String> = sortedSetOf()
```

Comment:
Unrelated to your change, but it reminds me that we're able to change `sortedSetOf<String>()` to `sortedSetOf()` also ni the line below. Mind doing that in a preceding commit in this PR?